core.coffee

```diff
@@ -36,6 +36,59 @@
 		'Options without $ prefixes (e.g. `options: filter: ...`) are deprecated, please use a $ prefix (eg `options: $filter: ...`).'
 	)
 
+	class Poll
+		channels:
```

Comment:
Needed a second thought to realize that this holds the `subscribers`. Would prefer to see it named like this (if more people agree) but no strong opinion.